modules/classroom/manifests/agent/git.pp

```diff
@@ -41,7 +41,7 @@
       recurse => true,
       require => [File[$sshpath],Exec['generate_key'],User['Administrator']],
     }
-    windows_env { 'C:\Program Files (x86)\Git\bin': }
+    windows_env { 'PATH=C:\Program Files (x86)\Git\bin': }
```

Comment:
Yes, else you won't be able to just type the command git in powershell and have it work.